README.md

```diff
@@ -21,3 +21,14 @@ $ pip install -r requirements.txt
 ### and now actually run it!
 $ python main.py brave-karaf 0.1.2 --gpg-key BB67A050 --git-hash 3cf4ac6577eb0d4775d20f24814e7a0852fa1635
 ```
+
+## Hacking
+
+Install linter, type-checker, and code formatter; run them before committing:
+
+```bash
+$ cd src
+$ . venv/bin/activate
+$ pip install -r requirements-dev.txt
+$ make
```

Comment:
Why arent't the previous steps automated in the Makefile? If you have a `venv` target that depends on `requirements-dev.txt` you can make so that the virtualenv is automatically rebuilt if the dependencies update.

In the Makefile you can then invoke the commands like `venv/bin/flake8` and that'd als remove the need to source the virtualenv. (not 100% sure that'll work, you might need `venv/bin/python -m flake8` you'll need to try)

Reply:
Historical reasons :D The instructions existed before the `Makefile`. Good idea though, I'll do do this in a separate PR.